workers/unity/Assets/link.xml

```diff
@@ -0,0 +1,20 @@
+<linker>
+	<assembly fullname="Improbable.Gdk.Core">
```

Comment:
we will have to actually add some documentation on our side to make sure that users are aware that we are doing this

Reply:
What would be a good way to document this?

Reply:
if you look at our repo, we have a docs folder. Create an internal google docs where you would add the documentation for this feature and then share it with the team so that we can review it before it will be added to the pr